src/board.c

```diff
@@ -28,18 +28,58 @@
 #include "osdvar.h"
 #include <math.h>
 
+
 void vTaskHeartBeat(void *pvParameters);
 void vTask10HZ(void *pvParameters);
 void triggerVideo(void);
 void triggerPanel(void);
 void checkDefaultParam(void);
 
+// Intra-task communication semaphores
+extern xSemaphoreHandle onScreenDisplaySemaphore;
+extern xSemaphoreHandle onMavlinkSemaphore;
+extern xSemaphoreHandle onUAVTalkSemaphore;
```

Comment:
These predate my changes -- this is original code.  I defined *mutexes*, not semaphores. FreeRTOS uses xSemaphoreTake on mutexes, so the confusion is understandable -- everything starts to look the same.

I did move things around, but nothing changed functionally, and the names are the original names.